kickstarts/partials/post/node.ks.erb

```diff
@@ -0,0 +1,5 @@
+# Get the NodeJS RPM repo setup
+curl --silent --location https://rpm.nodesource.com/setup_6.x | bash -
```

Comment:
I think typically the reason to avoid `curl <url> | bash -` is security as @Fryguy said.

This just also breaks from the pattern that we've established in general. If we could figure out what [this rpm](https://github.com/nodesource/distributions/blob/master/rpm/setup_6.x#L128) installs (my guess is a `.repo` file) then that would just solve it I think.